#### Problem Statement 19.2:
Using the following data, perform a oneway analysis of variance using α=.05. Write up
the results in APA format.

[Group1: 51, 45, 33, 45, 67]

[Group2: 23, 43, 23, 43, 45]

[Group3: 56, 76, 74, 87, 56]

In [2]:
Group1 = [51, 45, 33, 45, 67]

Group2 = [23, 43, 23, 43, 45]

Group3= [56, 76, 74, 87, 56]

** H0: Mean of Group1, Group2 and Group3 are equal <br>
H1: Mean of Group1, Group2 and Group3 are not equal **

In [3]:
import scipy.stats as stat

** Level of significance = p = 0.05 **

In [4]:
stat.f_oneway(Group1, Group2,Group3)

F_onewayResult(statistic=9.7472055030094626, pvalue=0.0030597541434430556)

**Conclusion:**

From the above ANOVA test since p<0.05, we reject the null hypothesis

df_error = 15-3 =12
df_groups = 3-1 = 2

**APA writeup**

F(2, 12)=9.75, p <0.05, η2=0.62.

In [ ]:
# from pyvttable import DataFrame

In [8]:
# from pivottablejs import pivot_ui